src/server/lib/signin/webhook.js

```diff
@@ -0,0 +1,37 @@
+import { randomBytes } from "crypto";
```

Comment:
Exact same as `src/server/lib/signin/email.js` but recieves an extra value (`(attribute, value, provider, options)` instead of `(email, provider, options)`)